dist/kongfig.js

```diff
@@ -66,8 +66,9 @@
 
 	var argv = __webpack_require__(321)(process.argv.slice(2));
 
-	if (!argv.path) {
-	    throw new Error('--path to the config file is required');
+	if (!_fs2['default'].existsSync(argv.path)) {
```

Comment:
I think this should be two checks one to check if --path was set and one to check if it was set that it is a valid path